src/providers/tvl/provider.ts

```diff
@@ -12,8 +12,9 @@ const indices = [
   "0x55b2CFcfe99110C773f00b023560DD9ef6C8A13B", // cdETI
   "0x72e364F2ABdC788b7E918bc238B21f109Cd634D7", // MVI
   "0x1494CA1F11D487c2bBe4543E90080AeBa4BA3C2b", // DPI
-  "0x0B498ff89709d3838a063f1dFA463091F9801c2b", // ETH2x-FLI
-  "0xAa6E8127831c9DE45ae56bB1b0d4D4Da6e5665BD", // BTC2x-FLI
+  "0x65c4C0517025Ec0843C9146aF266A2C5a2D148A2", // ETH2x
+  "0xD2AC55cA3Bbd2Dd1e9936eC640dCb4b745fDe759", // BTC2x
```

Comment:
These should be updated too right? (and replaced as opposed to added, as that'd double-count the TVL)